deps/wazuh_testing/wazuh_testing/tools/performance/statistic.py

```diff
@@ -1,25 +1,36 @@
+
+
+
```

Comment:
```suggestion
```

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/pull/4970/commits/8128e8a9a2702d28ba4216603d8539104506be48